pkg/okteto/secrets.go

```diff
@@ -139,6 +143,27 @@ func (c *userClient) GetContext(ctx context.Context, ns string) (*types.UserCont
 	return result, nil
 }
 
+// GetSecrets returns the secrets from Okteto API
+func (c *userClient) GetUserSecrets(ctx context.Context) ([]types.Secret, error) {
```

Comment:
Can we add a unit test for this new function?

Reply:
done c9e2b7deea5b348a5f9cddef298219dc33be523e